build.wake

```diff
@@ -1,3 +1,13 @@
+global topic chisel3Root: String
+def chisel3Root =
+  def paths = subscribe chisel3Root
+  match (head paths)
+    Some path  =
+      if (len paths) != 1 then
+        panic "Multiple published paths for chisel3 root: {format paths}"
+      else
+        path
+    None = "chisel3" # default path in workspace
```

Comment:
I think this would be cleaner as a direct pattern match rather than calling `head` on the list, since this is almost a textbook case of when pattern matching works best.

```suggestion
  match paths
    Nil = "chisel3" # default path in workspace
    path, Nil = path
    _ = panic "Multiple published paths for chisel3 root: {format paths}"
```